README.md

```diff
@@ -11,6 +11,8 @@ To make the first steps in the world of Merge easy, we've prepared this boilerpl
 
 All the components are built with React.js and styled with Emotion. Styles are using tokens stored in JSON files.
 
+To learn more about Merge – visit [our wiki](https://wikiuxpin.atlassian.net/wiki/spaces/MA/overview)
+
 ## Installation
```

Comment:
I think the `## Tests` section is in a wrong place – I'd put it at the bottom of the readme, or at least under "Adding new components", as it's a detail important for repository maintainers, not a users experimenting with the technology.